doc/manuals/orion-api.md

```diff
@@ -1,127 +1,81 @@
-
-## Conformance
-
-This specification describes the "full" compliance level.
-
-## Conventions
-
-NGSI version 2 uses camel case (camelCase) syntax for naming properties and related artifacts used
-by the API. When referring to URIs, as part of HATEOAS patterns, and to mark them appropriately,
-the suffix `_url` is added.
-
-## Reference Implementations
-
-* NGISv2 Context Brokers
-  * [Orion Context Broker](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker) - [Implementation Notes](https://fiware-orion.readthedocs.io/en/master/user/ngsiv2_implementation_notes/index.html)
+<!-- TOC -->
+
+- [Specification](#specification)
+    - [Introduction](#introduction)
+    - [Terminology](#terminology)
+        - [Context data modelling and exchange](#context-data-modelling-and-exchange)
+            - [Context Entities](#context-entities)
+            - [Context Attributes](#context-attributes)
+            - [Context Metadata](#context-metadata)
+    - [MIME Types](#mime-types)
+    - [JSON Entity Representation](#json-entity-representation)
+    - [JSON Attribute Representation](#json-attribute-representation)
+    - [Simplified Entity Representation](#simplified-entity-representation)
+    - [Partial Representations](#partial-representations)
+    - [Special Attribute Types](#special-attribute-types)
+    - [Builtin Attributes](#builtin-attributes)
+    - [Special Metadata Types](#special-metadata-types)
+    - [Builtin Metadata](#builtin-metadata)
+    - [Field syntax restrictions](#field-syntax-restrictions)
+    - [Attribute names restrictions](#attribute-names-restrictions)
+    - [Metadata names restrictions](#metadata-names-restrictions)
+    - [Ordering Results](#ordering-results)
+    - [Error Responses](#error-responses)
+    - [Geospatial properties of entities](#geospatial-properties-of-entities)
+        - [Simple Location Format](#simple-location-format)
+        - [GeoJSON](#geojson)
+    - [Simple Query Language](#simple-query-language)
+    - [Geographical Queries](#geographical-queries)
+        - [Query Resolution](#query-resolution)
+    - [Filtering out attributes and metadata](#filtering-out-attributes-and-metadata)
+    - [Notification Messages](#notification-messages)
+    - [Custom Notifications](#custom-notifications)
+    - [Group API Entry Point](#group-api-entry-point)
+        - [Retrieve API Resources [GET /v2]](#retrieve-api-resources-get-v2)
+- [API Routes](#api-routes)
+    - [Entities Operations](#entities-operations)
+        - [Entities List](#entities-list)
+            - [List Entities [GET /v2/entities{?limit,offset,options,type,id,idPattern,typePattern,q,mq,georel,geometry,coords,attrs,metadata,orderBy}]](#list-entities-get-v2entitieslimitoffsetoptionstypeididpatterntypepatternqmqgeorelgeometrycoordsattrsmetadataorderby)
```

Comment:
Let's simplify section names removing the list of parameters (the explanation of parameters will be in the section itself). 

I mean just:

```suggestion
            - [List Entities (`GET /v2/entities`)](#list-entities-get-v2entities)
```

Reply:
done in https://github.com/telefonicaid/fiware-orion/pull/4142/commits/dc7d5dcefa4f8f8a432adae2265cc6c6cb1f663c and https://github.com/telefonicaid/fiware-orion/pull/4142/commits/462fb895e30498c2d42fe25906bb0efbaf0bd701